class/defaults.yml

```diff
@@ -102,6 +105,10 @@ parameters:
         registry: quay.io
         image: appuio/oc
         tag: v4.13
+      forgejo:
+        registry: ghcr.io
+        image: vshn/forgejo
+        tag: latest
```

Comment:
this will probably restart instances once new version is available, we should pin to the latest symver.